exercise-3/styles.css

```diff
@@ -0,0 +1,65 @@
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.wrapper {
+  background-color: #413E3C;
+  display: flex;
+  flex-direction: row;
+  height: 30rem;
+  justify-content: center;
+  margin-top: 5rem;
+  width: 75rem;
+}
+
+input {
+  position: relative ;
+  top: 25rem;
+}
+
+labellll {
+  display: none;
+  position: fixed;
+}
+
+label {
+  padding-top: 5rem;
+  position: absolute;
+  left: -200rem;
+}
+
+input:checked + label {
+  display: block;
+  transition: 0.25s;
+  left: 31rem;
+}
+
+input:after {
```

Comment:
Unfortunately, `input` elements don’t get pseudo-elements. At least, not according to the spec. This doesn’t work in Firefox 😞. Fine for the exercise, but just FYI for later!

<img width="1200" alt="screen shot 2018-06-07 at 6 48 13 am" src="https://user-images.githubusercontent.com/1369770/41100652-2e2d5946-6a1f-11e8-8b38-609f820cf570.png">

Reply:
:( womp womp. I'll take a look at how you did it!